.circleci/config.yml

```diff
@@ -88,7 +88,10 @@ jobs:
           command: |
             # Load shell helpers (e.g. sdkman)
             source /home/fossa/.bashrc
-            # Run tests
+            # Run integration tagged native tests
+            go test ./... -tags=integration
```

Comment:
I've been running into some issues getting the tags to work as expected on the docker image, I'll take a longer look tomorrow